aspnetcore/toc.md

```diff
@@ -33,67 +32,89 @@
+
 ## [Advanced](mvc/advanced/index.md)
 ### [Working with the Application Model](mvc/controllers/application-model.md)
+### [Filters](mvc/controllers/filters.md)
+### [Areas](mvc/controllers/areas.md)
 ### [Application parts](mvc/advanced/app-parts.md)
 ### [Custom Model Binding](mvc/advanced/custom-model-binding.md)
-<!--### [🔧 Creating a Custom View Engine](mvc/advanced/custom-view-engine.md)-->
 ### [Custom formatters](mvc/advanced/custom-formatters.md)
+## [Tutorials](tutorials/index.md)
+
+### [Create a web app on Windows](tutorials/first-mvc-app/index.md)
+#### [Getting started](tutorials/first-mvc-app/start-mvc.md)
+#### [Adding a controller](tutorials/first-mvc-app/adding-controller.md)
+#### [Adding a view](tutorials/first-mvc-app/adding-view.md)
+#### [Adding a model](tutorials/first-mvc-app/adding-model.md)
+#### [Working with SQL Server LocalDB](tutorials/first-mvc-app/working-with-sql.md)
+#### [Controller methods and views](tutorials/first-mvc-app/controller-methods-views.md)
+#### [Adding Search](tutorials/first-mvc-app/search.md)
+#### [Adding a New Field](tutorials/first-mvc-app/new-field.md)
+#### [Adding Validation](tutorials/first-mvc-app/validation.md)
+#### [Examining the Details and Delete methods](tutorials/first-mvc-app/details.md)
+
+### [Create a web app on a Mac](tutorials/first-mvc-app/index.md)
+#### [Getting started](tutorials/first-mvc-app-mac/start-mvc.md)
+#### [Adding a controller](tutorials/first-mvc-app-mac/adding-controller.md)
+#### [Adding a view](tutorials/first-mvc-app-mac/adding-view.md)
+#### [Adding a model](tutorials/first-mvc-app-mac/adding-model.md)
+#### [Working with SQL Server LocalDB](tutorials/first-mvc-app-mac/working-with-sql.md)
+#### [Controller methods and views](tutorials/first-mvc-app-mac/controller-methods-views.md)
+#### [Adding Search](tutorials/first-mvc-app-mac/search.md)
+#### [Adding a New Field](tutorials/first-mvc-app-mac/new-field.md)
+#### [Adding Validation](tutorials/first-mvc-app-mac/validation.md)
+#### [Examining the Details and Delete methods](tutorials/first-mvc-app/details.md)
+
+### [Create a web app with VS Code](tutorials/first-mvc-app-xplat/index.md)
+#### [Getting started](tutorials/first-mvc-app-xplat/start-mvc.md)
+#### [Adding a controller](tutorials/first-mvc-app-xplat/adding-controller.md)
+#### [Adding a view](tutorials/first-mvc-app-xplat/adding-view.md)
+#### [Adding a model](tutorials/first-mvc-app-xplat/adding-model.md)
+#### [Working with SQL Server LocalDB](tutorials/first-mvc-app-xplat/working-with-sql.md)
+#### [Controller methods and views](tutorials/first-mvc-app-xplat/controller-methods-views.md)
+#### [Adding Search](tutorials/first-mvc-app-xplat/search.md)
+#### [Adding a New Field](tutorials/first-mvc-app-xplat/new-field.md)
+#### [Adding Validation](tutorials/first-mvc-app-xplat/validation.md)
+#### [Examining the Details and Delete methods](tutorials/first-mvc-app/details.md)
 
+### [Create a Web API](xref:tutorials/first-web-api-mac)
```

Comment:
Under MVC we now have Create a Web API and Building Web APIs.  I would put Create a web api under Building Web APIs, replacing the one that's there now.  Building will then have Create a web api, swagger, backend services